src/dos/dos_programs.cpp

```diff
@@ -95,6 +95,23 @@ void DOS_SetupPrograms(void)
 	MSG_Add("MSCDEX_UNKNOWN_ERROR","MSCDEX: Failure: Unknown error.\n");
 	MSG_Add("MSCDEX_WARNING_NO_OPTION", "MSCDEX: Warning: Ignoring unsupported option '%s'.\n");
 
+	MSG_Add("SHELL_CMD_RESCAN_HELP_LONG",
+	        "Clears the caches of a mounted drive.\n"
+	        "\n"
```

Comment:
Thinking about this from the users perspective - and what this does for them (as opposed to the internal mechanics).  Maybe:

```
Scans the DOS drive for changes made on the host.
Any changes will then be reflected inside DOS.
```

Reply:
It appears that if I put this on the introduction sentence then it will likely exceeds 80 characters. So I put it in "Notes" and rephrase it as "Changes to this drive made on the host will then be reflected inside DOS" so that no lines will exceed 80 characters.